pkg/logging/multihandler.go

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package logging
+
+import (
+	"context"
+	"log/slog"
+
+	"github.com/cilium/cilium/pkg/lock"
+)
+
+func NewMultiSlogHandler(handler slog.Handler) *multiSlogHandler {
+	return &multiSlogHandler{
+		mu:       lock.RWMutex{},
+		handlers: []slog.Handler{handler},
+	}
+}
+
+type multiSlogHandler struct {
+	mu       lock.RWMutex
+	handlers []slog.Handler
+}
+
+func (i *multiSlogHandler) Enabled(ctx context.Context, level slog.Level) bool {
+	i.mu.RLock()
+	defer i.mu.RUnlock()
+	for _, h := range i.handlers {
+		if h.Enabled(ctx, level) {
+			return true
+		}
+	}
+	return false
+}
+
+func (i *multiSlogHandler) Handle(ctx context.Context, record slog.Record) error {
+	i.mu.RLock()
+	defer i.mu.RUnlock()
+	for _, h := range i.handlers {
+		err := h.Handle(ctx, record)
+		if err != nil {
+			return err
+		}
```

Comment:
Should this terminate, meaning different handlers receive different output? Or collect errors and attempt to use the later handlers?